tasks/setup-RedHat.yml

```diff
@@ -4,6 +4,10 @@
     name: erlang
     state: present
 
+- name: Ensure legacy crypto policies
```

Comment:
This task (and the one below) would not be idempotent, I don't think—we need to make sure these only run if they need to be run...